src/app/model/Round.js

```
@@ -0,0 +1,50 @@
+import {Match} from './Match';
+
+export class Round {
+
+    constructor(numMatches) {
+        this._numMatches = numMatches;
+        this._matches = [];
+    }
+
+    get numMatches() {
+        return this._numMatches;
+    }
+
+    get matches() {
+        return this._matches;
+    }
+
+    get name() {
+        if (this.numMatches === 1) {
+            return 'Final Round';
+        } else {
+            return 'Round 1/' + this.numMatches;
+        }
+    }
+
+    addMatch(match) {
+        if (this._matches.length < this._numMatches) {
+            this._matches.push(match);
+        }
```

Never fail silently. If all matches were set, throw.